docs/guides/other_libs/serilog.md

```diff
@@ -0,0 +1,100 @@
+---
+uid: Guides.OtherLibs.Serilog
+title: Serilog
+---
+
+# Configuring serilog
+
+## Prerequisites
+
+- A basic working bot with a logging method as described in [Creating your first bot](xref:Guides.GettingStarted.FirstBot)
+
+## Installing the Serilog package
+
+You can install the following packages through your IDE or go to the nuget link to grab the dotnet cli command.
+
+|Name|Link|
+|--|--|
+|`Serilog.Extensions.Logging`| [link](https://www.nuget.org/packages/Serilog.Extensions.Logging)|
+|`Serilog.Sinks.Console`| [link](https://www.nuget.org/packages/Serilog.Sinks.Console)|
+
+## Configuring Serilog
+
+Serilog will be configured at the top of your async Main method, it looks like this
+
+```cs
+using Discord;
+using Serilog;
+using Serilog.Events;
+
+public class Program
+{
+  public static Task Main(string[] args) => new Program().MainAsync().GetAwaiter().GetResult();
```

Comment:
May I suggest removing the `public` keyword, and making this `void`, so the entry point of this app is like any default console application format in standard2.1+?